docker-helper

```diff
@@ -26,28 +26,66 @@ case $CMD in
     build)
         REPO="$2"
         BUILD_PATH="$3"
-        BRANCH=`echo $4 | sed 's/[\+\/]/-/g'`
-        COMMIT="$5"
-        BUILD_NUMBER="$6"
-        CI_TAG="$REPO:ci"
-        PROJ_NAME=$(basename $REPO)
-        GITHUB_REPO="docker.pkg.github.com/$REPO/$PROJ_NAME"
 
+        # Logic to determine the BRANCH tag
+        if [ -n "$4" ]; then
+            BRANCH=$(echo $4 | sed 's/[\+\/]/-/g')
+        else
+            BRANCH=$(echo $(git rev-parse --symbolic-full-name --abbrev-ref HEAD) | sed 's/[\+\/]/-/g')
```

Comment:
this might need a comment on why you are donig this. Also, I think you can simplify this line by doing 
```suggestion
            BRANCH=$(git rev-parse --symbolic-full-name --abbrev-ref HEAD | sed 's/[\+\/]/-/g')
```